### DISCUSSION TITLE
Should we restrict the value list for `relation/@name`?

### DISCUSSION THREAD

**Discussion by cmil** [dracor-org/dracor-schema#80] [Ideas]
The DraCor OpenAPI documentation restricts the `relation` query parameter of the spoken-text endpoint to a fixed list of values (see
https://github.com/dracor-org/dracor-api/blob/0a53edaa7ca64b9e10a16c6d4aa74bc5fc42a4c9/api.yaml#L852-L863). Now, https://github.com/dracor-org/dracor-api/issues/275 consistently suggests that the API should respond with an error if any other value than the defined ones is used. For utmost consistency we should then also restrict the value list for the `name` attribute of a `relation` element to the same list of values. 

Alternatively we could also follow our more recent approach and define a semi-fixed list of recommended values while leaving room for new values other parties might come up with. 

Please discuss, @lehkost, @ingoboerner !

**Comment by lehkost** [comment]
Values for relations are a fixed list, I think, we rely on the work of [Wiedmer et al. 2020](https://quadrama.github.io/publications/Wiedmer2020aa). There is currently no indication that anyone wants to expand values or enhance this modelling of relations.

**Reply by cmil** [reply]
So does that mean we want to treat that list as definite and enforce it in the schema for all current and future DraCor corpora? With the redesign of the schema we mostly moved away from strictly limiting how corpus editors can annotate things. To commit to an exclusive list of relation types without a technical need for it from the API perspective seems like a step backwards.

**Reply by ingoboerner** [reply]
The values that originally came from the above mentioned publication are also not fixed. If you look into GerDraCor-Coref, I think, they added additional relations already
